bot/otsu.go

```diff
@@ -54,36 +54,36 @@ func Otsu(wand *imagick.MagickWand, args OtsuArgs) ([]*imagick.MagickWand, error
 			return nil, fmt.Errorf("error writing colours back to image when converting to grayscale: %w", err)
 		}
 	}
-	var sum int
+	
```

Comment:
`go fmt` is showing changes on this line for me

Reply:
Should be fixed now